apps/state_mediator/test/state_mediator/integration/gtfs_test.exs

```diff
@@ -270,6 +270,7 @@ defmodule StateMediator.Integration.GtfsTest do
       assert [_] =
                ["CR-Lowell"]
                |> State.Shape.select_routes(1)
+               |> Enum.filter(&(&1.route_id == "CR-Lowell"))
```

Comment:
Why would it return shapes with other `route_id`'s? I'd expect this filter to be a noop, is it not the case? May be I'm missing something

Reply:
There are CR-Haverhill/CR-Lowell multi-route trips, so this includes them both.

@handorff Maybe we should include that as a part of the tests for those multi-route trips?